app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, foreign_key: :author_id
   belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes_counter + 1)
+  end
 end
```

Comment:
- [ ] It would be better if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to the database, then make the `update_likes_counter` method private :blush: